### PR TITLE
Stop execution in case of compilation errors

### LINKED ISSUE #25: Stop running output if compilation fails
Currently, when compilation fails, the program proceeds to the next step by displaying the output by running the `prog` file which is unnecessary. Therefore if program compilation fails simply display the errors and dont proceed to display program output.

### PR DESCRIPTION
**Pull request by Enigmage**
Closes #25

### CODE REVIEW

File: codemon/CodemonListen.py [hunk 1]
```diff
@@ -38,8 +38,8 @@ def isModified(event):
...
     print(colored.yellow('\nChange made at '+ filename))
     print(colored.cyan('\nCompiling '+ filename))
-    os.system('g++ ' + filename + ' -o ' + 'prog')
-    print('Running')	
-    print()
-    print(colored.yellow('Taking inputs from input.txt'))
-    os.system(f'{os.getcwd()}/prog < input.txt')
+    if os.system('g++ ' + filename + ' -o ' + 'prog') == 0:
+        print('Running')	
+        print()
```
Comment (ankushbhardwxj) [inline codemon/CodemonListen.py]: Could you get rid of this extra `print()` here ?

**Review by ankushbhardwxj — CHANGES_REQUESTED** [review]

File: codemon/CodemonListen.py (hunk 1 above)
Comment (Enigmage) [reply codemon/CodemonListen.py]: I did not add that extra print, it was already there. I can remove it if you want?

Comment (ankushbhardwxj) [reply codemon/CodemonListen.py]: It was added from a previous PR, you may remove it.

Comment (Enigmage) [reply codemon/CodemonListen.py]: okay, will do.

**Comment by Enigmage** [comment]
Made the requested change @ankingcodes

**Comment by ankushbhardwxj** [comment]
LGTM ! Thanks ! @Enigmage

**Pull request merged**